Assets/Plugins/CountlySDK/Services/UserDetailsCountlyService.cs

```diff
@@ -84,6 +71,21 @@ public async Task SetUserDetailsAsync(CountlyUserDetailsModel userDetailsModel)
                     throw new Exception("Accepted picture formats are .png, .gif and .jpeg");
                 }
 
+
+                userDetailsModel.Name = TrimValue(" '" + nameof(userDetailsModel.Name) + "' ", userDetailsModel.Name);

```

Comment:
All these " '" + nameof(userDetailsModel.Name) + "' " are just too wasteful. Totally unnecessary 21 string concatination, not counting the ones in the call + using "nameof" 7 times.

Just use a static sting value or precompute this resulting string at module init time.

Reply:
and these passed value names are just a copy paste thing. They were not updated after pasting